src/main/java/org/troyargonauts/Constants.java

```diff
@@ -2,9 +2,18 @@
 
 public final class Constants {
 
-    public interface Manipulator {
-        public static final int kForwardChannel = 0;
-        public static final int kReverseChannel = 1;
+    public interface IntakeConstants {
```

Comment:
The proper notation for constants file is not hungarian notation, it should be all uppercase with underscores as spaces. 
For example: kRightIntakeID would be RIGHT_INTAKE_ID.

Also, since the variables are in the Intake interface, there is no need for the word Intake.